### PR TITLE
fix: deprecated code paths on Symfony 5.3

### PR DESCRIPTION
**Pull request by chalasr**
| Q             | A
| ------------- | ---
| Branch?       | 2.6
| Tickets       | -
| License       | MIT
| Doc PR        | -

### CODE REVIEW

File: tests/Fixtures/TestBundle/Security/AbstractSecurityUser.php [hunk 1]
```diff
@@ -0,0 +1,34 @@
...
+if (interface_exists(PasswordAuthenticatedUserInterface::class)) {
+    abstract class AbstractSecurityUser implements UserInterface, PasswordAuthenticatedUserInterface
+    {
+        abstract public function getPassword(): ?string;
+
+        public function getUserIdentifier(): string
+        {
+            return $this->getUsername();
+        }
+    }
```
Comment (chalasr) [inline tests/Fixtures/TestBundle/Security/AbstractSecurityUser.php:29]: https://symfony.com/blog/new-in-symfony-5-3-improvements-for-security-users

File: tests/Fixtures/app/AppKernel.php [hunk 2]
```diff
@@ -138,7 +138,7 @@ protected function configureContainer(ContainerBuilder $c, LoaderInterface $load
 
         $alg = class_exists(NativePasswordHasher::class) || class_exists('Symfony\Component\Security\Core\Encoder\NativePasswordEncoder') ? 'auto' : 'bcrypt';
         $securityConfig = [
-            'encoders' => [
+            class_exists(NativePasswordHasher::class) ? 'password_hashers' : 'encoders' => [
```
Comment (chalasr) [inline tests/Fixtures/app/AppKernel.php:141]: https://symfony.com/blog/new-in-symfony-5-3-passwordhasher-component

**Review by alanpoulain — APPROVED** [review]
LGTM

**Review by dunglas — APPROVED** [review]

**Comment by alanpoulain** [comment]
Thank you @chalasr.

**Pull request merged**